pkgs/applications/blockchains/lightning-terminal/default.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  mkYarnPackage,
+  buildGoModule,
+  nodejs,
+  yarn,
+  gnumake,
+  go,
+  git,
+  fixup_yarn_lock
+}:
+
+buildGoModule rec {
+  pname = "lightning-terminal";
+  version = "0.12.4-alpha";
+  src = fetchFromGitHub {
+    owner = "lightninglabs";
+    repo = "lightning-terminal";
+    rev = "v${version}";
+    hash = "sha256-gGo9abd12zZPPTBkmEfB8BNK1ZVpea2uZNpeYRb5l00=";
+  };
+
+  app = mkYarnPackage {
+    name = "lightning-app";
+    src = "${src}/app";
+    packageJSON = "${src}/app/package.json";
+    yarnLock = "${src}/app/yarn.lock";
+  };
+
+  vendorHash = "sha256-MwVTmwOb0wW2e9lEbr/Pm2IhHXizwEgaw5j+x8+XA+E=";
+
+  ldflags = [
+    "-s" "-w"
+    "-X" "github.com/lightningnetwork/lnd/build.Commit=lightning-terminal-"
```

```suggestion
    "-X github.com/lightningnetwork/lnd/build.Commit=lightning-terminal-"
```

Also for the followings

(Also what's "lightning-terminal-")